pkg/libvirttools/virtualization.go

```diff
@@ -139,8 +140,9 @@ func (v *VirtualizationTool) processVolumes(mounts []*kubeapi.Mount, domXML stri
 	}
 
 	for _, mount := range mounts {
+		volumeName := containerName + "_" + strings.Replace(*mount.ContainerPath, "/", "_", -1)
```

Comment:
What if `mount.ContainerPath` is `nil`?

I'd suggest to do `mount.GetContainerPath()` - https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/api/v1alpha1/runtime/api.pb.go#L386-L391